### ISSUE TITLE
Verify impact of upgrading to Roslyn 1.1

### ISSUE THREAD

**Issue by giggio** [code-cracker/code-cracker#770]
- [x] Does it work with VS 2015 RTM (Not update 1) with nuget?
- [x] Does it work with VS 2015 RTM (Not update 1) with vsix?

Anything else?

**Comment by giggio** [comment]
@carloscds @ElemarJR Just finished testing on a VS 2015 RTM (Community Edition, on Azure). Neither Nuget nor vsix work. I created an analyzer project from scratch, published it with version 1.0, it worked.
Then I tried upgrading it to 1.1.1, it did not work anymore.
Same thing with version 1.2.1, it does not work.

This means that if we upgrade CodeCracker to use any of the latest Roslyn APIs it will stop working on VS 2015 RTM.
The only package that can be upgraded is `Microsoft.CodeAnalysis.Analyzers` to version `1.1.0`, which is exactly what comes on the the Roslyn template.
